src/java/org/apache/cassandra/index/sai/disk/v1/IndexWriterConfig.java

```diff
@@ -85,10 +90,10 @@ public class IndexWriterConfig
     private final int bkdPostingsMinLeaves;
 
     private final int maximumNodeConnections;
-
     private final int constructionBeamWidth;
-
     private final VectorSimilarityFunction similarityFunction;
+    private final VectorSourceModel sourceModel;
+
     private final OptimizeFor optimizeFor;
```

Comment:
Nit: we can get rid of this variable now.

Reply:
Not sure what you mean, we use this in COHG.

(Possible source of confusion: the constructor variable assigned to it is named defaultSourceModel, even though it is not always the default, I will fix that.)